lib/loginpage/loginpage.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+import './loginpage_form.dart';
+import 'loginpage_image.dart';
+
+import '../constants/constants.dart' as Constansts;
+
+class LoginPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final screenWidth = MediaQuery.of(context).size.width;
+    final screenHeight = MediaQuery.of(context).size.height;
+    return Scaffold(
+        body: Container(
+            height: double.infinity,
+            width: double.infinity,
+            decoration: const BoxDecoration(
+                image: DecorationImage(
+                    image: AssetImage("assets/gradient.png"),
```

Comment:
Please create a file for constants to define these paths

Reply:
Done